services/parser-database/connector.py

```diff
@@ -3,6 +3,7 @@
 # import requests  # pylint: disable=import-error
 import random
 import logging
+import numpy as np
```

Comment:
Does this need to be added to requirements.txt or is this a python standard library?

Reply:
I think it needs to be added